### ISSUE TITLE
Allow to set existing stage

### ISSUE THREAD

**Issue by maicol07** [EasyG0ing1/SceneOneFX#3]
Hi,
can you add a method to set a stage when SceneOne starts?
Unfortunately, this is missing from SceneOneFX and prevents Headless testing with TestFX.
Thanks

**Comment by EasyG0ing1** [comment]
The core class in SceneOneFX is technically a utility class and is never instantiated, and because of that, there isn't really a "start" to the class, merely the ability to add SceneObjects through the Builder class, which is accessed when you call
```
SceneOne.set(sceneId, Parent).options...build();
```
Give me an example of how you would need to interact with the class so that you can assign your stage, because there are a few ways that I could think of where it could be incorporated, such as assigning your stage to the class, then all scenes could be shown on that stage, or you could pass in a stage into the above "set" call where the SceneObject would simply use the stage you pass in as an argument ... but if I could see an example of what you would like to do (pseudo code that would help me understand the context) - that would be helpful.

**Comment by maicol07** [comment]
My use case would be to use the stage that TestFX gives me:

[https://github.com/maicol07/OOP21-knighttale/blob/933ef4402d58c8a67db629d509ba22a1be397e0b/src/test/java/it/unibo/aknightstale/views/MainMenuTest.java#L19](https://github.com/maicol07/OOP21-knighttale/blob/933ef4402d58c8a67db629d509ba22a1be397e0b/src/test/java/it/unibo/aknightstale/views/MainMenuTest.java#L19)

This is how TestFX gives me the stage, then I load FXML to SceneOne, so something before `.build()` to set the stage would be awesome.

**Comment by EasyG0ing1** [comment]
@maicol07 I'm going to assume that you're using maven, so download [this zip file](https://simtechdata.com/SceneOneFX.zip) and unzip it. 

Assuming your mvn command is in your path, just run either the win.bat file if you're in Windows or if in Mac/Linux run the nix.sh (though you might need to do a ```chmod 777 *``` first)

Then in your project, change your POM file dependency to this:
```XML
<dependency>
    <groupId>com.simtechdata</groupId>
    <artifactId>SceneOneFX</artifactId>
    <version>1.2.3-SNAPSHOT</version>
</dependency>
```

The you'll be able to do this
```Java
SceneOne.set(sceneId, Parent).stage(Stage).build();
```

SceneOneFX will use the stage you give it instead of creating a new Stage when you use the ```.stage()``` build statement.

If it works, let me know and I'll push it out as a release.

**Comment by maicol07** [comment]
> @maicol07 I'm going to assume that you're using maven, so download [this zip file](https://simtechdata.com/SceneOneFX.zip) and unzip it. 
> 
> Assuming your mvn command is in your path, just run either the win.bat file if you're in Windows or if in Mac/Linux run the nix.sh (though you might need to do a ```chmod 777 *``` first)
> 
> Then in your project, change your POM file dependency to this:
> ```XML
> <dependency>
>     <groupId>com.simtechdata</groupId>
>     <artifactId>SceneOneFX</artifactId>
>     <version>1.2.3-SNAPSHOT</version>
> </dependency>
> ```
> 
> The you'll be able to do this
> ```Java
> SceneOne.set(sceneId, Parent).stage(Stage).build();
> ```
> 
> SceneOneFX will use the stage you give it instead of creating a new Stage when you use the ```.stage()``` build statement.
> 
> If it works, let me know and I'll push it out as a release.

No, I'm using Gradle 😅

**Comment by EasyG0ing1** [comment]
@maicol07 Are you proficient enough with gradle to take the jar file in that zip and incorporate it into your project for testing? Because I don't know much about gradle, but worst case, you can just remove the gradle reference to the library and manually add the jar file to your project in your IDE.

**Comment by EasyG0ing1** [comment]
@maicol07 [This link](https://stackoverflow.com/questions/20700053/how-to-add-local-jar-file-dependency-to-build-gradle-file) might be of some help.

**Comment by EasyG0ing1** [comment]
Resolved in version 1.2.3
